astacus/common/ipc.py

```diff
@@ -221,6 +221,7 @@ class CassandraRestoreSSTablesRequest(NodeRequest):
     table_glob: str
     keyspaces_to_skip: Sequence[str]
     match_tables_by: CassandraTableMatching
+    expect_empty_target: bool
```

Comment:
It looks like this message is also used in the normal restore case, not only the delta, won't that break compatibility of nodes of different versions ?

I guess that can be OK, since all nodes in a restore are supposed to be recent, unless the creation of a cluster perfectly overlaps with the deployment of a new astacus version ?

Reply:
> unless the creation of a cluster perfectly overlaps with the deployment of a new astacus version ?
that'd have to be very perfect timing, and even if so - the cluster is not operational yet, we can simply drop the old nodes and deploy fresh ones

Reply:
Yes, that's quite unlikely, looks fine then.